src/components/forums/MyComments.js

```diff
@@ -0,0 +1,126 @@
+import NavigationBar from '../common/NavigationBar';
+import Box from "@material-ui/core/Box";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const MyComments = () => {
+
+    const { currentUser } = useAuth();
+    const history = useHistory();
+    const [email, setEmail] = React.useState('');
+    const [userID, setUserID] = React.useState('');
+
+    const [myComments, setMyComments] = React.useState([]);
+
+    React.useEffect(() => {
+        setEmail(currentUser.email);
+        loaduserSearchByEmail(currentUser.email);
+        //loadGetForums(); 
+    }, []);
+
+    const loaduserSearchByEmail = (email) => {
+        callApiGetuserSearchByEmail(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetuserSearchByEmail = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        //console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                email: email
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    useEffect(() => {
+        loadGetCommentsByUserID();
+    }, [userID]);
+
+    const loadGetCommentsByUserID = async () => {
+        try {
+            const res = await CallApiGetCommentsByUserID();
+            const parsed = res.express;
+            setMyComments(parsed);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    const CallApiGetCommentsByUserID = async () => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/getForumCommentsByUserID`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                userID: userID
+            })
+        });
+        const body = await response.json();
+        console.log("got here");
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    My Comments
+                </Typography>
+            </Box>
+
+    <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+        {myComments.map((comment) => (
```

Comment:
should the comments render from newest to oldest? currently, it is rendering from oldest to newest

Reply:
Not quite sure, as the events, forums and forum comments currently render from oldest to newest. I believe that the sudden switch may feel a bit confusing to the user.

Reply:
Disregard! There is no current order - will be fixed thanks :)